src/frontend/apps/lti_site/components/VideoWidgetProvider/widgets/DownloadVideo/index.spec.tsx

```diff
@@ -123,4 +129,99 @@ describe('<InstructorDownloadVideo />', () => {
     const downloadButton = screen.getByRole('button', { name: 'Download' });
     expect(downloadButton).toBeDisabled();
   });
+
+  it('check toggle allow download', async () => {
+    const mockedVideo = videoMockFactory();
+    const cache = createIntlCache();
+    const intl = createIntl(
+      {
+        locale: 'en',
+        messages: {},
+      },
+      cache,
+    );
+    const {
+      allowDownloadToggleLabel,
+      allowDownloadToggleFail,
+      allowDownloadToggleSuccess,
+      disallowDownloadToggleSuccess,
+    } = messages;
```

Comment:
This is not what we use to do. We always use the string directly and we should continue to do that. Thanks